abi.html

```diff
@@ -5587,9 +5579,67 @@ <h4><a href="#closure-types">5.1.8 Closure Types (Lambdas)</a></h4>
 href="#mangle.lambda-sig">lambda-sig</a>&gt; can only ever refer to a
 template parameter of a generic lambda.
 
+<p>
+<a name="lexical-ordering">
+<h4><a href="#lexical-ordering">5.1.9 Lexical ordering</a></h4>
+
+<p>
+Lexical ordering is used for numbering local entities
+(unnamed classes and enumerations,
+closure types, and static local variables)
+when there is no other way of distinguishing them.
+Except as described below, all local entities are to be numbered,
+even if subsequent optimization makes some of them unnecessary,
+or no mangled name is actually required for some of them.
+The ordering of entities appearing in a
+mem-initializer-list shall be the order that the entities appear in
```

Comment:
FIXME: Clarify that the order is the order in which the entities *end*:

```
auto f() { return [x = []{}]{ static int n; return &n; }(); } // x is the 1st lambda, n is within the 2nd lambda
```

Reply:
Pushed a suggested reworking of the new section to account for what I think the right model is.